stressor/stress-test.mts

```diff
@@ -323,79 +408,179 @@ console.log(
+      console.log(`| ${formatter(key)} | ${totalRuns} | ${totalUnequal} | ${(totalUnequal * 100 / totalRuns).toFixed(2)}% |`);
+    }
+  }
+
+  generateSummary('Test Plan', result => result.workflowTestPlan);
+  generateSummary('AT', result => result.workflowId, workflowIdAsLabel);
+  generateSummary('Browser', result => result.workflowBrowser);
+
+  console.log(`\n## Summary by All\n`);
+  console.log(`| Test Plan | AT | Browser | Total Tests | Unequal Responses | Unequal % |`);
+  console.log("| --- | --- | --- | --- | --- | --- |");
+  for (const combo of keys) {
+    const comboResults = results.get(combo);
+    // typescript insists this is possibly undefined
+    if (comboResults) {
+      console.log(`| ${comboResults.workflowTestPlan} | ${workflowIdAsLabel(comboResults.workflowId)} | ${comboResults.workflowBrowser} | ${comboResults.totalRows} | ${comboResults.unequalRows} | ${comboResults.percentUnequal.toFixed(2)}% |`)
+    }
+  }
+
+  const formatResponses = (responses: Array<string>, newlineTab: string = '\n    '): string =>
+    responses
+      .map((response, index) => {
+        const responseWrapped = response.split('\n').map(line => wrap(line, {width: 60, newline: newlineTab})).join(`\n${newlineTab}`);
+        return `Response ${index+1}:${newlineTab}${responseWrapped}`;
+      })
+      .join('\n')
+      .replace(/\n(\s*\n)+/g, '\n');
+
+  for (const combo of keys) {
+    const comboResults = results.get(combo);
+    // typescript insists this is possibly undefined
+    if (comboResults) {
+      console.log(`\n## ${combo.workflowTestPlan} ${workflowIdAsLabel(combo.workflowId)} ${combo.workflowBrowser}\n`);
+      console.log(`\n### Run Logs\n`);
+      let logNumber = 0;
+      for (const url of comboResults.logUrls) {
+        console.log(`* [Run #${logNumber++}](${url})`);
+      }
+      for (const comparedResult of comboResults.comparedResults) {
+        console.log(`\n### Test Number: ${comparedResult.testCsvRow}\n`);
+        console.log(`__${combo.workflowTestPlan} ${workflowIdAsLabel(combo.workflowId)} ${combo.workflowBrowser}__`);
+        console.log(`#### Most Common Responses:`);
+        console.log("```");
+        console.log(formatResponses(comparedResult.baselineResponses));
+        console.log("```");
+        for(const diverges of comparedResult.differences) {
+          console.log(`#### Divergent responses from [Run ${diverges.runId}](${comboResults.logUrls[diverges.runId]}):`);
+          console.log("```diff");
+          console.log(diff(formatResponses(comparedResult.baselineResponses), formatResponses(diverges.responses)));
+          console.log("```");
+        }
+      }
+    }
+    console.log(``)
+  }
+};
+
+formatResultsForMD(allResults);
+
+// Debug helper: write the needed "allResults" for this run to a json file
+import { writeFile } from "node:fs/promises";
+await writeFile("stressor-run.json", JSON.stringify([...allResults.entries()]), "utf-8");
```

Comment:
I think it's worth leaving this in regardless, but we might want to enable it with command line flag.  It's super useful when working on little formatting things to be able to just run it on the last data.